upd_script/update_all.sh

```diff
@@ -522,18 +522,18 @@ autodetect_setup
 
 # Update Cinch, if it's installed.
 # check for file /home/pi/cinch, if it is, call cinch setup.
-if [ ! -f /home/pi/cinch ]; then
-    echo "No Cinch Found."
-else
-    feedback "Found cinch, running Cinch install."
-    cd $RASPBIAN_PATH/upd_script/wifi
-    sudo ./cinch_setup.sh
-fi
+#if [ ! -f /home/pi/cinch ]; then
+#    echo "No Cinch Found."
+#else
+#    feedback "Found cinch, running Cinch install."
+#    cd $RASPBIAN_PATH/upd_script/wifi
```

Comment:
So, after running DI Update both via terminal & via GUI app, the Cinch was still functional. So I haven't seen any bug here.